roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -104,10 +104,11 @@
 
 - name: Stop if ip var does not match local ips
   assert:
-    that: ip in ansible_all_ipv4_addresses
+    that: (ip in ansible_all_ipv4_addresses) or (ip in ansible_all_ipv6_addresses)
     msg: "'{{ ansible_all_ipv4_addresses }}' do not contain '{{ ip }}'"
   when:
     - not ignore_assert_errors
+    - (ansible_all_ipv4_addresses | length>0) or (ansible_all_ipv6_addresses | length>0)
```

Comment:
I guess this new line 111 change is unnecessary for IPv6 only environments because the new line 107 seems enough for solving the issue.

Reply:
> I guess this new line 111 change is unnecessary for IPv6 only environments because the new line 107 seems enough for solving the issue.

Done